.cspell/other.txt

```diff
@@ -8,3 +8,6 @@ myapp
 TMPDIR
 mktemp
 coreutils
+signalfx
+SIGNALFX
```

Comment:
this one should not be needed (already handled by the previous line)

Reply:
removed in https://github.com/signalfx/splunk-otel-dotnet/pull/124/commits/ced068fa5c04eb6458594eb103331c0a0f7d3f13